src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -400,12 +422,34 @@ public async Task<IActionResult> UpdateDisputeAsync(string guidHash, [FromBody]
                 return NotFound("Dispute not found");
             }
 
-            var givenNames = response.Message.DisputantGivenName1 + " " + response.Message.DisputantGivenName2 + " " + response.Message.DisputantGivenName3;
-            if (response.Message.DisputantSurname != user?.Surename
-                || !(response.Message.DisputantGivenName1 == user?.GivenName || givenNames.TrimEnd() == user?.GivenNames))
-            {
-                return BadRequest("Disputant not match");
-            }
+            // TODO: Uncomment this before User Acceptance Testing

```

Comment:
instead of commenting out code, you could just wrap in a if (false). Otherwise the code does not compile and any changes will not be updated here and will cause more edits to be required when this needs to be uncommented.

You can also add a `#warning Disputant name comparison has been disabled` so it shows up in the build warnings. This can help prevent this from being overlooked when moving to production.

Reply:
done in upcoming commit